$NetBSD: patch-configure,v 1.1 2021/12/11 16:00:59 gdt Exp $

Kludge around use of bash extensions.

Reported upstream via email.

--- configure.orig	2021-03-17 08:13:26.000000000 +0000
+++ configure
@@ -17956,10 +17956,9 @@ then
 $as_echo_n "checking GPERF version... " >&6; }
     GPERF_VERSION=`"$GPERF" --version | head -n1 2>&1 | sed 's,GNU gperf ,,'`
     case $GPERF_VERSION in
-      1.*) GPERF_VERSION=$((100+${GPERF_VERSION:2:1})) ;;
-      2.*) GPERF_VERSION=$((200+${GPERF_VERSION:2:1})) ;;
-      3.*) GPERF_VERSION=$((300+${GPERF_VERSION:2:1})) ;;
-      4.*) GPERF_VERSION=$((400+${GPERF_VERSION:2:1})) ;;
+      3.0) GPERF_VERSION=300 ;;
+      3.1) GPERF_VERSION=301 ;;
+      4.*) GPERF_VERSION=400 ;;
     esac
     { $as_echo "$as_me:${as_lineno-$LINENO}: result: $GPERF_VERSION" >&5
 $as_echo "$GPERF_VERSION" >&6; }
